packages/dev/s2-docs/src/MobileHeader.tsx

```diff
@@ -120,27 +136,37 @@ export function MobileHeader({toc, pages, currentPage}) {
         animationTimeline: 'scroll()',
         animationRange
       } as CSSProperties}>
-      <div
-        className={style({
-          display: 'flex',
-          gap: 12,
-          alignItems: 'center',
-          flexGrow: 1
-        })}>
-        {icon}
-        <h2
+      <div className={style({flexGrow: 1})}>
+        <Link
+          href={homepage}
```

Comment:
I also fixed this in https://github.com/adobe/react-spectrum/pull/9220 so we'll have conflicts